### PR TITLE
Fixed Rapidoid's benchmark meta-data.

### PR DESCRIPTION
**Pull request by nmihajlovski**
This PR fixes a problem introduced by @ashawnbandy-te-tfb with commit 88e374bccee0dbc84881005449463b12359ed868, which prevented Rapidoid's tests from executing.

The approach is also reverted back to "Realistic".

### CODE REVIEW

**Comment by korri123** [comment]
Might have forgot to fix the display name, it was changed from rapidoid-http-fast to Rapidoid in that commit too

**Comment by korri123** [comment]
Might have forgot to fix the display name, it was changed from rapidoid-http-fast to Rapidoid in that commit too

File: frameworks/Java/rapidoid/benchmark_config.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
   "framework": "rapidoid",
   "tests": [{
     "default": {
-      "setup_file": "setup",
+      "setup_file": "setup-low-level",
```
Comment (msmith-techempower) [inline frameworks/Java/rapidoid/benchmark_config.json]: This is fine.

File: frameworks/Java/rapidoid/benchmark_config.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
       "json_url": "/json",
       "plaintext_url": "/plaintext",
       "port": 8080,
-      "approach": "Stripped",
+      "approach": "Realistic",
```
Comment (msmith-techempower) [inline frameworks/Java/rapidoid/benchmark_config.json]: This is not.

Comment (nmihajlovski) [inline frameworks/Java/rapidoid/benchmark_config.json]: Why that's not fine?
The code has already been refactored exactly for this purpose.

Comment (nmihajlovski) [inline frameworks/Java/rapidoid/benchmark_config.json]: The code has been merged already on master branch.

**Comment by nmihajlovski** [comment]
@msmith-techempower @ashawnbandy-te-tfb The deadline for round-13-related fixes is approaching, and this PR hasn't been merged yet. If something is preventing you from merging it, how can I help?

**Comment by ashawnbandy-te-tfb** [comment]
@nmihajlovski We'll be starting another preview run shortly with this change merged in.

**Comment by nmihajlovski** [comment]
Thanks!

**Pull request merged**